ceagle/api/v1/security.py

```diff
@@ -14,14 +14,28 @@
 #    under the License.
 
 import flask
+from flask import request
 
+from ceagle.api import client
+from ceagle.api_fake_data import base as base_fake
+from ceagle.api_fake_data import fake_security
 
 security = flask.Blueprint("security", __name__)
 
+if base_fake.USE_FAKE_DATA:
```

Comment:
This is yet another approach of Fake API implementation, there is another one already via decorator, example: https://github.com/seecloud/ceagle/blob/89c8e430232c54bf5f51d81ff2784962e31aad07/ceagle/api/v1/status.py#L27

It is very doubtful that having two approaches is good choice, only one should be is use